docs/Project.toml

```diff
@@ -1,5 +1,57 @@
+name = "DiffEqFlux"
+uuid = "aae7a2af-3d4f-5e19-a356-7da93b79d9d0"
+authors = ["Chris Rackauckas <accounts@chrisrackauckas.com>"]
+version = "1.12.1"
+
 [deps]
-Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+Adapt = "79e6a3ab-5dfb-504d-930d-738a2a938a0e"
+BlackBoxOptim = "a134a8b2-14d6-55f6-9291-3336d3ab0209"
+DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+DiffEqSensitivity = "41bf760c-e81c-5289-8e54-58b1f1f8abe2"
+DiffResults = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
+Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
+ForwardDiff = "f6369f11-7733-5829-9624-2563aa707210"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
+Optim = "429524aa-4258-5aef-a3af-852621145aeb"
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+ProgressLogging = "33c8b6b6-d38a-422a-b730-caa89a2f386c"
+RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
+Requires = "ae029012-a4dd-5104-9daa-d747884805df"
+ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
+Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
+ZygoteRules = "700de1a5-db45-46bc-99cf-38207098b444"
 
 [compat]
-Documenter = "0.24.2"
+Adapt = "1"
+BlackBoxOptim = "0.5"
+DiffEqBase = "6.36"
+DiffEqSensitivity = "6.19"
+DiffResults = "0.0.4, 1.0"
+Flux = "0.10.1"
+ForwardDiff = "0.10"
+Optim = "0.20, 0.21"
+ProgressLogging = "0.1"
+RecursiveArrayTools = "2"
+Requires = "0.5, 1.0"
+ReverseDiff = "1.1"
+StaticArrays = "0.11, 0.12"
+Tracker = "0.2"
+ZygoteRules = "0.2"
+julia = "1"
+
+[extras]
+DelayDiffEq = "bcd4f6db-9728-5f36-b5f7-82caef46ccdb"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+MultistartOptimization = "3933049c-43be-478e-a8bb-6e0f7fd53575"
+NLopt = "76087f3c-5699-56af-9a33-bf431cd00edd"
+OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
+SafeTestsets = "1bc83da4-3b8d-516f-aca4-4fe02f6d838f"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+StochasticDiffEq = "789caeaf-c7a9-5a7d-9973-96adeb23e2a0"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
I think you accidentally changed the Project.toml in the docs

Reply:
Yeah. Dumb question: should I keep the Project.toml as it's in my repo and add the dependencies there or I should copy paste the Project.toml in DiffEqFlux and add some lines to include Distributions.jl?

Reply:
revert this and change the one in the top level.